primitives/primproc/columncommand.cpp

```diff
@@ -463,44 +544,20 @@ void ColumnCommand::processResult()
     }
 }
 
+// Used by PseudoCC at least
 void ColumnCommand::createCommand(ByteStream& bs)
 {
     uint8_t tmp8;
-    uint32_t tmp32;
-
     bs.advance(1);
+    colType.unserialize(bs);
```

Comment:
Could you please move Command serialization/deserialization closer to each other? Maybe it makes sense to create separate functions for this purpose, and in ColumnCommand and ColumnCommandJL just call them.

Reply:
Decided to do this refactoring separately.